mipengine/controller/controller_logger.py

```diff
@@ -1,14 +1,37 @@
 import logging
+from mipengine.controller import config as ctrl_config
 
 
-def getRequestLogger():
+def get_request_logger(context_id):
     """
     Used for logging information produced after an endpoint request.
     """
-    return logging.getLogger("controller_request")
+    logger = (
```

Comment:
Remove "controller_request" logger initialization completely from the controller `__init__` file.